airflow/utils/log/es_task_handler.py

```diff
@@ -53,9 +53,12 @@ class ElasticsearchTaskHandler(FileTaskHandler, LoggingMixin):
     PAGE = 0
     MAX_LINE_PER_PAGE = 1000
 
+    # 16 is reasonable in this case
+    # pylint: disable-msg=too-many-arguments
     def __init__(self, base_log_folder, filename_template,
                  log_id_template, end_of_log_mark,
                  write_stdout, json_format, json_fields,
+                 index='*',
```

Comment:
Just to check `*` will match all indexes? I am working with an ES cluster where we have a new index every day

Reply:
yes